test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -57,7 +58,8 @@ static inline void multi_consumer_pub_sub_test(bool intra_process)
   rclcpp::executors::MultiThreadedExecutor executor;
   // Try to saturate the MultithreadedExecutor's thread pool with subscriptions
   for (uint32_t i = 0; i < executor.get_number_of_threads(); i++) {
-    auto sub = node->create_subscription<test_rclcpp::msg::UInt32>(node_topic_name, 16, callback, callback_group);
+    auto sub = node->create_subscription<test_rclcpp::msg::UInt32>(node_topic_name, 16, callback,
+        callback_group);
```

Comment:
Please wrap after the open parenthesis instead of in the middle of the args. Same below.